cli/cmd/Deploy.go

```diff
@@ -141,6 +141,8 @@ var deployCmd = &cobra.Command{
 			elapsed := time.Since(start)
 			fmt.Printf("Deploy successful, took %v\n", elapsed)
 		}
+
+		api.NotifyGrafanaAboutDeploy(strings["app"], strings["cluster"], strings["namespace"], strings["version"])
```

Comment:
Should not be needed as the `deploy` cmd is just sending a http request to naisd deploy

Reply:
Agree. This should be server side only